bika/lims/browser/listing/ajax.py

```diff
@@ -383,7 +384,18 @@ def set_field(self, obj, name, value):
 
         # field exists, set it with the value
         if field:
-            obj.edit(**{field.getName(): value})
+            # N.B. We don't use the `edit` method here to bypass the instance
+            #      permission check for `Modify portal content`.
+            # obj.edit(**{field.getName(): value})
```

Comment:
Would you mind to add the PR's url here?. Sometimes is helpful to see all involved changes to fully understand why this is done like this. Specially interesting in this case, in which we are bypassing `Modify portal content` permission check.